Common/Data/Custom/SEC/SECReportFiler.cs

```diff
@@ -30,8 +30,8 @@ public class SECReportFiler
         /// <summary>
         /// Information regarding the filing itself
         /// </summary>
-        [JsonProperty("FILING-VALUES")]
-        public SECReportFilingValues Values;
+        [JsonProperty("FILING-VALUES"), JsonConverter(typeof(SingleValueListConverter<SECReportFilingValues>))]
```

Comment:
👍 - Do we want a test of this?

Reply:
I think we should create some custom tar archive with a few test cases to cover these changes. Good call, will get to work on that